dotCMS/src/test/java/com/dotmarketing/business/cache/provider/h22/H22CacheTest.java

```diff
@@ -100,24 +122,75 @@ public void testInit() throws Exception {
 		cache.removeAll();
 		assertThat("Cache flushed, we have no groups", cache.getGroups().size() == 0);
 
-
+    }
 		
+    @Test
+    public void test_multithreaded_access() throws Exception {
+
+        // test dumping and rebuilding cache
+        //testMultithreaded(cache, numberOfThreads,true, false, true);
+        
+        // test causing an error and recovering
+        testMultithreaded(false, false);
+
+    }
+    
+    
+    
+    
+    
+    
+    @Test
+    public void test_breaking_cache_access() throws Exception {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5796487)